EVENT.md

```diff
@@ -858,10 +859,10 @@ browsers.
 ### Reporting cooldown / rate limits
 
 To limit the amount of user identity leakage between a <source site,
-destination> pair, the browser should throttle the amount of total information
+ site> pair, the browser should throttle the amount of total information
```

Comment:
Should we still include destination here?

Reply:
Yep, good catch. Done.